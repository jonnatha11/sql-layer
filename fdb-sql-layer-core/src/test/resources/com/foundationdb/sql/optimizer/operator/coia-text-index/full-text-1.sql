SELECT * FROM customers WHERE FULL_TEXT_SEARCH(name, '"john smith"')