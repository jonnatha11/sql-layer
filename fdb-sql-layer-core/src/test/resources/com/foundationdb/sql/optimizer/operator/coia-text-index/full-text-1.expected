PhysicalSelect[cid:int, name:varchar(32)]
  Project_Default(customers.cid, customers.name)
    GroupLookup_Default(customers -> customers)
      IndexScan_FullText(test.customers.cust_ft, name:"john smith")